subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/AbstractInitIntegrationSpec.groovy

```diff
@@ -53,12 +57,20 @@ abstract class AbstractInitIntegrationSpec extends AbstractIntegrationSpec {
         targetDir.file(".gitignore").assertIsFile()
         targetDir.file(".gitattributes").assertIsFile()
     }
+
     protected void commonJvmFilesGenerated(BuildInitDsl scriptDsl) {
         commonFilesGenerated(scriptDsl)
         subprojectDir.file("src/main/resources").assertIsDir()
         subprojectDir.file("src/test/resources").assertIsDir()
     }
 
+    protected void mavenCentralRepositoryDeclared(BuildInitDsl scriptDsl) {
+        assertThat(subprojectDir.file(scriptDsl.fileNameFor("build")).text, containsString("mavenCentral()"))
```

Comment:
We should really consider using assertj at some point instead of hamcrest. Maybe not as part of this change :)